injected/src/content-feature.js

```diff
@@ -214,7 +217,7 @@ export default class ContentFeature {
         })
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
```

Comment:
Did we drop these checks?

Reply:
`@typescript-eslint/no-unused-vars` is now [allowed for function arguments](https://github.com/duckduckgo/eslint-config/blob/main/index.js#L148). I'm not sure why it was flagged before because [it's in Standard](https://github.com/standard/eslint-config-standard/blob/v17.1.0/.eslintrc.json#L176).

`@typescript-eslint/no-empty-function` is not flagged on this line even in the main branch.